apsuite/orbcorr/orbit_correction.py

```diff
@@ -121,23 +125,30 @@ def correct_orbit(self, jacobian_matrix=None, goal_orbit=None):
 
         for _ in range(self.params.maxnriters):
             dkicks = -1*_np.dot(ismat, dorb)
-            kicks = self._process_kicks(dkicks)
+            kicks, saturation_flag = self._process_kicks(dkicks)
             self.set_kicks(kicks)
             orb = self.get_orbit()
             dorb = orb - goal_orbit
             figm = OrbitCorr.get_figm(dorb)
             diff_figm = _np.abs(bestfigm - figm)
             if figm < bestfigm:
                 bestfigm = figm
+            if saturation_flag:
+                return OrbitCorr.CORR_STATUS.Convergence_fail
```

Comment:
I think the name of the `CORR_STATUS` associated with this fail should be named `Saturation_fail` or something like that.